pyroclast/selfboosting/modules.py

```diff
@@ -1,4 +1,4 @@
-import sonnet as snt
+#import sonnet as snt
```

Comment:
Here, I think I'm going to tell you to eat my shorts. This file should stay here, commented because it's useful reference, and because I'm not sure if anyone will even remember it exists if we delete it. I'll let it drag on a little longer and we can delete it in the future if we decide that's the thing to do.